File: UIPresentation/UI/src/app/models/payment-request-summary-models/payment-request-summary.ts
import { Time } from "@angular/common"
import { Timestamp } from "rxjs"
import { StatusEnum } from "src/app/enums/status-enum.enum"

export class PaymentRequestSummary {
    id!:number
    paymentRequestId!:number
    status!:StatusEnum
    uploadDate!:Date
    quantity!:number
    amount!:number
    userId!:number
    systemEnteredDate!:Date
    systemEnteredTime!:Time
    systemEnteredRegistrationNumber!:number
    isActive!:boolean
}

File: Entities/Concrete/Entities/PaymentRequestSummary.cs
﻿using Core.Entities.Abstract;
using Entities.Concrete.Enums;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Entities.Concrete.Entities
{
    public class PaymentRequestSummary : IEntity
    {
        public int Id { get; set; }
        public int PaymentRequestId { get; set; }
        public StatusEnum Status { get; set; }
        public DateTime UploadDate { get; set; }
        public int Quantity { get; set; }
        public decimal Amount { get; set; }
        public int UserId { get; set; }
        public DateTime SystemEnteredDate { get; set; }
        public TimeSpan SystemEnteredTime { get; set; }
        public int SystemEnteredRegistrationNumber { get; set; }
        public DateTime Optime { get; set; }
        public bool IsActive { get; set; }
 
    }
}
